test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.Protocol.cs

```diff
@@ -565,6 +565,62 @@ public class Protocol
                     await connection.DisposeAsync().OrTimeout();
                 }
             }
+
+            [Fact]
+            public async Task ClientPingsAutomatically()
+            {
+                var connection = new TestConnection();
+                var hubConnection = CreateHubConnection(connection);
+
+                hubConnection.TickRate = TimeSpan.FromMilliseconds(30);
+                hubConnection.PingInterval = TimeSpan.FromMilliseconds(80);
+
+                try
+                {
+                    await hubConnection.StartAsync().OrTimeout();
+                    await Task.Delay(120);
```

Comment:
I don't think we should have this

Reply:
Yeah, that's a good point, the `ReadSentTextMessageAsync` call will block until the ping message arrives, so there's no need for the `Task.Delay` :)